spec/airbrake/user_informer_spec.rb

```diff
@@ -4,6 +4,7 @@
 SingleCov.covered! uncovered: 8 # if defined?(Rack) block is not tested
 
 describe Airbrake::UserInformer do
+  before { Airbrake.user_information_placeholder = "<!-- AIRBRAKE ERROR -->" }
```

Comment:
don't set it ... but add a custom test that changes in ... and then unsets it in the `ensure`